ibis/mapd/tests/test_client.py

```diff
@@ -0,0 +1,365 @@
+import collections
+
+from datetime import date, datetime
+
+import pytest
+
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+
+import ibis
+import ibis.common as com
+import ibis.expr.datatypes as dt
+import ibis.expr.types as ir
+
+
+pytestmark = pytest.mark.mapd
+pytest.importorskip('pymapd')
+
+
+def test_literal_execute(client):
+    expected = '1234'
+    expr = ibis.literal(expected)
+    result = client.execute(expr)
+    assert result == expected
+
+'''
+def test_simple_aggregate_execute(alltypes, df):
```

Comment:
Many of these tests are BigQuery specific. Can you remove them from here?

The preferred alternative is to add appropriate tests in `ibis/tests/all/test_*.py`. To do that, you'll also need to add a `MapD` class in `ibis/tests/all/backends.py`. There are many examples in that file to get you started.